cli/src/cmd/forge/verify.rs

```diff
@@ -177,6 +228,41 @@ impl VerifyArgs {
         Ok(())
     }
 
+    async fn verify_with_sourcify(self) -> eyre::Result<()> {
+        let url = format!(
+            "https://sourcify.dev/server/check-by-addresses?addresses={}&chainIds={}",
+            hex::encode(self.address.as_bytes()),
```

Comment:
this formats with 0x prefix, note debug formatting is necessary here
```suggestion
            format!("{:?}", self.address),
```